releng/org.eclipse.xtext.contributor/Xtext.setup

```diff
@@ -767,10 +767,10 @@
             name="org.eclipse.xtext.sdk.feature.group"/>
         <requirement
             name="org.objectweb.asm"
-            versionRange="[6.0.0,7.0.0)"/>
+            versionRange="[6.1.1,7.0.0)"/>
```

Comment:
I would prefer to work with a fixed version.

Reply:
i dont want to do this now. can you create an issue at eclipse/xtext so we can discuss this in general and fix it between the beta and release